deploy/docker/start.sh

```diff
@@ -46,6 +43,12 @@ while [ $IDLE_TIME -lt 5 ]; do
     IDLE_TIME=$(expr $IDLE_TIME + 1)
     if curl http://localhost:${MGMT_PORT}/status >/dev/null 2>&1; then
         IDLE_TIME=0
+        # Print the latest erlang.log
+        erlang_logs=$(ps -ef |grep "tail -f /opt/emqx/log/erlang.log" |grep -v grep | sed -r "s/.*tail -f (.*)/\1/g")
+        new_erlang_log=$(echo $(ls -t /opt/emqx/log/erlang.log.*) | awk '{print $1}')
```

Comment:
```suggestion
        new_erlang_log="$(ls -t /opt/emqx/log/erlang.log.* | head -1)"
```

And we should always quote `"$(...)"`.
btw, if we run shellcheck on this script, there is going to be quite many errors :)

Reply:
I had a hard time following why there was a need to `echo` `$(ls -t ...)`,
Now I might have guessed what's the intention here: `ls` prints files as tabular (multi lines). 
However, it's only tabular when the output is a terminal (with width), otherwise always one file each line,
when `ls` output is piped to another command, it's always newline as delimiter. 
You can test it by piping `ls` to a file: `ls > a` then inspect the file.